MediaBrowser.Api/UserLibrary/ItemsService.cs

```diff
@@ -227,6 +228,16 @@ private QueryResult<BaseItem> GetQueryResult(GetItems request, DtoOptions dtoOpt
                 request.IncludeItemTypes = "Playlist";
             }
 
+            if (!user.Policy.EnabledFolders.Where(i => new Guid(i).Equals(item.Id)).Any() && !user.Policy.EnableAllFolders)
```

Comment:
* IMO reversing the checks seems more logical.
* Use the equality operator instead of `.Equals`
* Move the lambda from the `Where` into the `Any`